### PR TITLE
Documentation: tweak GSG

### PR DESCRIPTION
**Pull request by wenlingz**
* Remove the older kernel version(pk414)
* Add a one-line command to insert the PARTUUID into the acrn.conf
  file (without copying it all manually)

### CODE REVIEW

**Comment by wenlingz** [comment]
@gvancuts please also help review this PR which is the your original PR 2533, thanks.

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -124,7 +124,6 @@ partition. Follow these steps:
       kernel-org.clearlinux.native.4.20.2-683
       kernel-org.clearlinux.iot-lts2018-sos.4.19.13-1901141830
```
Comment (gvancuts) [inline doc/getting-started/apl-nuc.rst]: Which Clear Linux version are we using as a reference here?

I suspect the `release` field of the kernel is of the new type (much shorter) so the kernel number looks like `kernel-org.clearlinux.iot-lts2018-sos.4.19.19-8`

Comment (wenlingz) [reply doc/getting-started/apl-nuc.rst]: We will update one reference here, thanks

Comment (wenlingz) [reply doc/getting-started/apl-nuc.rst]: @gvancuts done, please help check, thanks

**Pull request merged**